tests/integration/kubernetes/k8s-sealed-secret.bats

```diff
@@ -86,17 +85,19 @@ setup() {
 
 
 @test "Unseal Env Secrets with CDH" {
-	[ "${KATA_HYPERVISOR}" = "qemu-coco-dev" ] || skip "Test not ready yet for ${KATA_HYPERVISOR}"
+	kbs_set_resource "default" "sealed-secret" "test" "unsealed_secret"
+	k8s_create_pod "${K8S_TEST_ENV_YAML}"
 
-	if [ "${KBS}" = "false" ]; then
-		skip "Test skipped as KBS not setup"
-	fi
+	kubectl logs secret-test-pod-cc
+	kubectl logs secret-test-pod-cc | grep -q "UNPROTECTED_SECRET = not_sealed_secret"
+	kubectl logs secret-test-pod-cc | grep -q "PROTECTED_SECRET = unsealed_secret"
+}
 
+@test "Unseal File Secrets with CDH" {
 	kbs_set_resource "default" "sealed-secret" "test" "unsealed_secret"
-	k8s_create_pod "${K8S_TEST_YAML}"
+	k8s_create_pod "${K8S_TEST_FILE_YAML}"
 
 	kubectl logs secret-test-pod-cc
-	kubectl logs secret-test-pod-cc | grep -q "UNPROTECTED_SECRET = not_sealed_secret"
```

Comment:
Any reason to not test a non-sealed secret in a a file?

Reply:
Added the test about non-sealed secret